translation.json

```diff
@@ -2028,6 +2028,126 @@
+    {
+      "name": "Dutch",
+      "code": "nl",
+      "translation": {
+        "AppName": "Tiny File Manager",
+        "AppTitle": "Bestand beheerder ",
+        "Login": "Inloggen",
+        "Username": "Gebruikersnaam",
+        "Password": "Wachtwoord",
+        "Logout": "Uitloggen",
+        "Copy": "Kopiëren",
+        "Save": "Opslaan",
+        "SelectAll": "Alles selecteren",
+        "UnSelectAll": "Alles deselecteren",
+        "File": "Bestan",
+        "Back": "Terug",
+        "Size": "Grootte",
+        "Perms": "Rechten",
+        "Modified": "Aangepast",
+        "Owner": "Eigenaar",
+        "Search": "Zoeken",
+        "NewItem": "Nieuw item",
+        "Folder": "Map",
+        "Delete": "Verwijder",
+        "Rename": "Hernoem",
+        "CopyTo": "Kopiëren naar",
+        "DirectLink": "Directe link",
+        "UploadingFiles": "Bestanden opladen",
+        "ChangePermissions": "Rechten aanpassen",
+        "Copying": "Bezig met kopiëren",
+        "CreateNewItem": "Maak nieuw item aan",
+        "Name": "Naam",
+        "AdvancedEditor": "Geavanceerde bewerker",
+        "RememberMe": "Onthoud mij",
+        "Actions": "Acties",
+        "Upload": "Opladen",
+        "Cancel": "Annuleren",
+        "InvertSelection": "Keer selectie om",
+        "DestinationFolder": "Doelmap",
+        "ItemType": "Item type",
+        "ItemName": "Item naam",
+        "CreateNow": "Onmiddelijk aanmaken",
+        "Download": "Download",
+        "Open": "Openen",
+        "UnZip": "Uitpakken",
+        "UnZipToFolder": "Uitpakken in folder",
+        "Edit": "Aanpassen",
+        "NormalEditor": "Standaard bewerker",
+        "BackUp": "Back-up aanmaken",
+        "SourceFolder": "Bronmap",
+        "Files": "Bestanden",
+        "Move":"Verplaatsen",
+        "Change": "Aanpassen",
+        "Settings": "Instellingen",
+        "Language": "Taal",
+        "Folder is empty":"Map is leeg",
+        "PartitionSize":"Partitiegrootte",
+        "ErrorReporting":"Foutmeldingen",
+        "ShowHiddenFiles":"Toon onzichtbare bestanden",
+        "Full size":"Volledige groote",
+        "Help":"Hulp",
+        "Free of":"Maak vrij van",
+        "Preview":"Voorbeeld",
+        "Help Documents":"Hulp documenten",
+        "Report Issue":"Probleem melden",
+        "Generate":"Genereer",
+        "FullSize":"Volledige groote",
+        "FreeOf":"Maak vrij van",
+        "CalculateFolderSize":"Bereken map groote",
+        "ProcessID":"Proces-ID",
+        "Created":"Aangemaakt",
+        "HideColumns":"Verberg Rechten/Eigenaar kollomen",
+        "You are logged in":"U ben ingelogd",
+        "Check Latest Version":"Bekijk laatste versie",
+        "Generate new password hash":"Genereer een nieuw wachtwoord hash",
+        "Login failed. Invalid username or password":"Inloggen mislukt. Verkeerd wachtwoord of gebruikersnaam",
+        "password_hash not supported, Upgrade PHP version":"password_hash is niet ondersteund, Upgrade PHP versie",
+
```

Comment:
remove empty line

Reply:
The empty line was a divider of the new and the old stuff. thought it would be useful.